SwiftSearch: Add version 7.5.1

cvladan opened the pull request:
Homepage: https://swiftsearch.sourceforge.io/

Licence: Creative Commons Attribution Non-Commercial License V2.0 (CC BY-NC)
Licence identifier from [SPDX](https://spdx.org/licenses/) is CC-BY-NC-2.0.

I hope everything is properly prepared.
I've done a lot of research and **this is still the best tool for the job**. 
If you are interested, check out: https://notes.cvladan.com/treasury/windows/windows-searching-for-files/.

Author: Mehrdad N.
Hash: sha256

<!--
  By opening this PR you confirm that you have searched for similar issues/PRs here already.
  Failing to do so will most likely result in closing of this PR without any explanation.
  It is also mandatory to open a relevant issue (either Package Request or Bug Report) for
  discussion with the maintainers, before creating any new PR.
  Read the contributing guide first to save both your and our time.
-->

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

cvladan committed:
Please add amazing SwiftSearch tool

Homepage: https://swiftsearch.sourceforge.io/

Licence: Creative Commons Attribution Non-Commercial License V2.0 (CC BY-NC)
Licence identifier from [SPDX](https://spdx.org/licenses/) is CC-BY-NC-2.0.

I hope everything is properly prepared.
I've done a lot of research and **this is still the best tool for the job**.
If you are interested, check out: https://notes.cvladan.com/treasury/windows/windows-searching-for-files/.

Author: Mehrdad N.
Hash: sha256

cvladan committed:
Merge branch 'ScoopInstaller:master' into patch-1

cvladan committed:
Fix hash field

I guess this is now a valid hash field value.

HUMORCE reviewed (changes requested):
An incomplete manifest.

- `architecture`, this app has 64bit and 32bit
- download url (static)
- checkver and autoupdate

cvladan committed:
Merge branch 'ScoopInstaller:master' into patch-1

cvladan committed:
Update swiftsearch.json

Fixed

cvladan committed:
Update swiftsearch.json

Added "rename-fragment" to auto-update URL

cvladan commented:
1. _Architecture_
   The linked file is both 32bit and 64bit and I believe that it is most flexible to include that version, as file itself is already small and there is no strong reason to introduce separate URL's.

1. _Static URL_
   Fixed

1. Checkver and Autoupdate
   Fixed both

cvladan committed:
Merge branch 'ScoopInstaller:master' into patch-1
